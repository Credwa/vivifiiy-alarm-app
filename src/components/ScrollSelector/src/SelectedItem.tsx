import React, { FC } from 'react';
import { View, StyleSheet } from 'react-native';

interface IProps {
  itemHeight: number;
}

const SelectedItem: FC<IProps> = ({ children, itemHeight }) => (
  <View style={{ ...styles.selectedItem, height: itemHeight }}>{children}</View>
);

export default SelectedItem;

const styles = StyleSheet.create({
  selectedItem: {
    height: 30,
    justifyContent: 'center',
    alignItems: 'center'
  }
});
