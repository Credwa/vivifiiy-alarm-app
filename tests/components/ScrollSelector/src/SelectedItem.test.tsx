import React from 'react';
import renderer from 'react-test-renderer';
import SelectedItem from '@/components/ScrollSelector/src/SelectedItem';

describe('ScrollSelector/src/ <SelectedItem />', () => {
  it(`renders correctly`, () => {
    const tree = renderer.create(<SelectedItem key={0} itemHeight={60} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
