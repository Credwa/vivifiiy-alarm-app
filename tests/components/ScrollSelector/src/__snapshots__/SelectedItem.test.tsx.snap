// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScrollSelector/src/ <SelectedItem /> renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "height": 60,
      "justifyContent": "center",
    }
  }
/>
`;
